ui/cypress/fixtures/kubernetes/storageclasses.json

```diff
@@ -0,0 +1,207 @@
+{
+  "apiVersion": "storage.k8s.io/v1",
+  "items": [
```

Comment:
I would limit the storage classes only to the one that makes sense for our tests

Reply:
OK. I was hesitating should I keep only one storage class. I will do so then.